src/framework/auth/services/auth.spec.ts

```diff
@@ -24,7 +24,7 @@ describe('auth-service', () => {
   let tokenService: NbTokenService;
   let dummyAuthStrategy: NbDummyAuthStrategy;
   const testTokenValue = 'test-token';
-  const ownerStrategyName = 'strategy';
+  const ownerStrategyName = 'dummy';
```

Comment:
do we need this change? looks unrelated

Reply:
Yes we do since i added `isAuthenticatedOrRefresh()` calls in the test units.
The strategy provided in test module is named 'dummy'.
So i modified the ownerStrategyName to `dummy` and replaced all 'dummy' occurrences by `ownerStrategyName`.
I know everything is not related to this PR, but is was necessary.